news/2019-03-31-beatmap-spotlights-winter-2018.md

```diff
@@ -0,0 +1,308 @@
+
+Despite the harsh yet minimalist nature of the song, the map uses stack-based jumpstreams to emphasize held guitar notes and repeated 1/2 instruments, trills for repeated 1/4 instruments, and 1/4 long notes to bring out higher pitched synths to take advantage of every musical phrase. By utilizing repetition, these concepts combine to create air-tight consistency that ensures that pattern significance and musical relevancy is kept throughout the entire map in a way that is difficult yet extremely exciting for players.
+
+All in all the map provides an unorthodox approach to jumpstreams that will hopefully inspire future mappers interested in pushing the envelope on jumpstream patterning in the ranked section!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/mania/destination.jpg)](https://osu.ppy.sh/beatmapsets/843723)
+
+His first mapset in the ranked section, [Dergo](https://osu.ppy.sh/users/7439226) throws a giant surprise with some of the most unique qualities in a mapset you could see in ranked. Though succducc is somewhat well known in standard, this particular track is not very well known. The choices that were taken into mapping are unusually spectacular - the first half of the song being one large buildup is represented well through pattern complexity, and the kiai's usage of jacks, bursts, and trills is not something normally seen in ranked. 
+
+The balance in this map between serenity and chaos perfectly represents the music it accompanies. For this reason, the mapset has a clearly well-deserved spot in the spotlights, and we will definitely expect more from Dergo in the future.
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/mania/relentless.jpg)](https://osu.ppy.sh/beatmapsets/881771)
+
+Are you ready for the ruffneck bass? As a skillful osu!mania mapper and Beatman Nominator who brought the ranked section with a few delicate 4K spreads in the past, [_Stan](https://osu.ppy.sh/users/1653229) this time returns with an unexpected 7K marathon of this exciting piece of electronic music, DJ Noriken - #The_Relentless(Modified), that testifies to his deep understanding of 7K mapping as well.
+
+This marathon is mostly composed of normal notes, but in a way that fully convey every single line of music, making it actually more interesting and beginner-friendly. However, it is challenging at the same time, since _Stan creates it in a way that gradually poses more difficulty onto the map. At the last kiai, the full mixture of short LNs with streams greatly enhances the overall difficulty of the map to mark the pinnacle of his understanding over the progression of the music, while it also necessarily avoids SR inflation.
```

Comment:
```suggestion
This marathon is mostly composed of normal notes, but in a way that fully convey every single line of music, making it actually more interesting and beginner-friendly. However, it is challenging at the same time, since \_Stan creates it in a way that gradually poses more difficulty onto the map. At the last kiai, the full mixture of short LNs with streams greatly enhances the overall difficulty of the map to mark the pinnacle of his understanding over the progression of the music, while it also necessarily avoids SR inflation.
```